src/app/authorization/login/login.component.scss

```diff
@@ -0,0 +1,120 @@
+@use '../../../../public/assets/styles/variables' as *;
+@use '../../../../public/assets/styles/reset' as *;
+
+a{
+    color: $link-btn-color;
+}
+button{
+    color: $third-font-color;
+     background-color: $link-btn-color;
+}
+.login {
```

Comment:
To avoid directly selecting class names we can start creating a `nested `structure like:
```
.login{  
  .login-card{  
    //And same for each login classes be nested inside login parent
  } 
```